avocado/utils/process.py

```diff
@@ -510,8 +528,8 @@ def _fill_streams(self):
         # Clean subprocess pipes and populate stdout/err
         self._popen.stdout.close()
         self._popen.stderr.close()
-        self.result.stdout = self.get_stdout()
-        self.result.stderr = self.get_stderr()
+        self.result.stdout_bytes = self.get_stdout()
+        self.result.stderr_bytes = self.get_stderr()
```

Comment:
Yes, we do. For example iso9660 reads arbitrary binary data from isoinfo/isoread. Likewise, you can imagine tests running e.g. `pnmtopng` using `avocado.utils.process`.